examples/rabbitmq-consumer-with-client-acknowledgement/rabbitmq-consumer-with-client-acknowledgement.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on manually acknowledging the messages consumed from a RabbitMQ queue using Ballerina.
+keywords: ballerina, ballerina by example, rabbitmq, consumer, bbe, listener, service, client, ack, acknowledgment
```

Comment:
The best practice is to add around 5 - 8 keywords, I updated the template doc with that info as well (missed adding that earlier). Shall we remove some?